### ISSUE TITLE
Keep repeater when rescheduling multiple tasks

### ISSUE THREAD

**Issue by ilian2233** [orgzly-revived/orgzly-android-revived#163]
- I have searched for existing issues that may be the same as or related to mine.

First of all, very cool app!

My suggestion is when rescheduling multiple todos and only one of them has a `repeater`, `time`, or `delay` for the `repeater/time/delay` to not be deleted.
So basically rescheduling multiple todos to change only the `day`.

**Comment by stefan2904** [comment]
I agree that this would be useful. Even if all the items had a repeater, the repeater would currently get lost (if not manually re-added).

**Comment by ngm** [comment]
Yes please, I sorely miss this.  If rescheduling multiple by a day in Emacs you keep the repeaters.
